cleanlab/classification.py

```diff
@@ -263,6 +263,7 @@ def fit(
         noise_matrix=None,
         inverse_noise_matrix=None,
         label_issues=None,
+        sample_weight=None,
```

Comment:
I'm not sure we want to add `sample_weight` as an extra required argument of `fit()`. This makes this class no longer applicable to users' own bespoke models whose `fit()` function does not take in this argument.

Instead note that you can already pass in kwargs into the `clf_kwargs` argument. So training with `sample_weight` is already supported for `clf.fit()` during cross-validation. It is currently just not supported during the final `clf.fit()` call on the cleaned subset of data, but you could easily support it (see my tips below on how to do this).